AirCastingTests/SearchAndFollowTests/AirBeamMeasurementDownloaderTests.swift

```diff
@@ -0,0 +1,49 @@
+// Created by Lunar on 03/08/2022.
+//
+import XCTest
+import Resolver
+@testable import AirCasting
+
+class AirBeamMeasurementDownloaderTests: ACTestCase {
+    lazy var sut = AirBeamMeasurementsDownloaderDefault()
+    let clientSpy = APIClientSpy()
+    
+    func test_whenRequestingDataFromServer_shouldUseAssertedPath() {
+        Resolver.test.register { self.clientSpy as APIClient }
+        sut.downloadStreams(with: Int.default, completion: { _ in })
+        guard let url = URLComponents(string: clientSpy.request.description)?.url else { XCTFail("Problem when trying to get url."); return }
+        XCTAssertEqual(url.path, "/api/fixed/sessions/\(Int.default)/streams.json")
+    }
+    
+    func test_whenRequestingDataFromServer_shouldUseGetMethod() {
+        Resolver.test.register { self.clientSpy as APIClient }
+        sut.downloadStreams(with: Int.default, completion: { _ in })
```

Comment:
See - here using `default` is perfect because streamID is not participating in what we're testing, so we're telling the reader not to worry about the value.